### PR TITLE
Mention IKS's one-click install process

### PR DESCRIPTION
**Pull request by duglin**
Signed-off-by: Doug Davis <dug@us.ibm.com>

### CODE REVIEW

**Comment by RichieEscarez** [comment]
Thanks for the contribution!

From just a quick scan of the change, i noticed a typo but more importantly, this documentation should probably have two sections to make clear that there are two separate paths.

For example use an H2 title, "Installing Knative using an IKS managed add-on" and another H2 title: "Manually installing Knative on IKS" (where both have their own Before you begin sections, in which is where you can explain how to get the add-on before the steps on how to use it).

**Comment by sebgoa** [comment]
I agree with Richie.

I would modify the intro a tidbit to have a bullet list with the two install method (add-on and manual) and then make the add-on install on-par with the manual using the same header...

otherwise /lgtm

We should really think about extracting the common install steps that are in all the install guides.

**Comment by sebgoa** [comment]
Oh...and you probably want to move to 0.5.0

**Comment by duglin** [comment]
sorry for the delay - see what you think about the updates.

File: docs/install/Knative-with-IKS.md [hunk 1]
```diff
@@ -131,7 +125,28 @@ components, the recommended configuration for a cluster is:
...
+
+The easiest way to install it is using the Managed Knative add-on facility.
+This one-click install process will install Knative, and Istio if not already
+installed, and provide automatic updates and lifecycle management of your
+Knative control plane.
+
+You can get the add-on via the "Add-ons" tab of your Kubernetes cluster's
+console page, or via the command line:
+```bash
+bx ks cluster-addon-enable -y knative $CLUSTER_NAME
```
Comment (sebgoa) [inline docs/install/Knative-with-IKS.md]: how do we get `bx` CLI ?

File: docs/install/Knative-with-IKS.md [hunk 2]
```diff
@@ -131,7 +125,28 @@ components, the recommended configuration for a cluster is:
...
+```
+
+For more information about the add-on see
+[here](https://cloud.ibm.com/docs/containers?topic=containers-knative_tutorial#knative_tutorial).
+
+## Manually installing Knative on IKS 
+
```
Comment (sebgoa) [inline docs/install/Knative-with-IKS.md]: I would add a little intro sentence here :)

File: docs/install/Knative-with-IKS.md (hunk 1 above)
Comment (duglin) [reply docs/install/Knative-with-IKS.md]: thanks for this!  it should be `ibmcloud` not `bx` (that's old).

File: docs/install/Knative-with-IKS.md (hunk 2 above)
Comment (duglin) [reply docs/install/Knative-with-IKS.md]: Done! see what you think.

**Pull request merged**